schematic/manifest/generator.py

```diff
@@ -1554,12 +1559,9 @@ def get_manifest(
 
             # TODO: Update or remove the warning in self.__init__() if
             # you change the behavior here based on self.use_annotations
-                            
-            # populate empty manifest with content from downloaded/existing manifest
-            manifest_sh = self.set_dataframe_by_url(empty_manifest_url, manifest_record[1])
 
             # determine the format of manifest
-            result = self._handle_output_format_logic(output_format = output_format, output_path = output_path, sheet_url = sheet_url, manifest_url = manifest_sh.url, empty_manifest_url=empty_manifest_url, dataframe = manifest_record[1])
+            result = self._handle_output_format_logic(output_format = output_format, output_path = output_path, sheet_url = sheet_url, empty_manifest_url=empty_manifest_url, dataframe = manifest_record[1])
```

Comment:
Nit: Personally, I think the logic can be improved which would simplify things a bit. 

1. Separate out the concerns
    1. If you want to return the dataframe -> Don't call this function at all
    1. You have 2 output formats: "excel", "sheet_url", one returns the Excel spreadsheet and one returns the sheet URL
1. Have 3 output formats: "excel", "dataframe", "sheet_url"

What I like about option 1 is...

1. You simplify the `_handle_output_format_logic`. It seems odd to pass in output_format as "dataframe" AND the dataframe only to just return the dataframe.
2. You simplify your function and the test you have to write for the function by just returning on return type (str)
3. You can remove `sheet_url`

What I like about option 2...
1. It might be the least change to the existing code
2. You can remove `sheet_url`

Reply:
Yeah. I agree with that technically, sheet_url is redundant. But Milen said that for keeping the code backward compatible, we should keep the option. I think maybe i could open a separate jira issue to clear this task.